### PR TITLE
Add instructions for running with Docker

### PR DESCRIPTION
**Pull request by smithclay**
Adds `docker-compose.yml` and instructions for running this plugin using Docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +114,14 @@ View the query you made in Grafana by clicking the **View query** button.
 ![View Grafana query](https://github.com/lightstep/lightstep-metrics-datasource/raw/main/images/docs/graf_metciViewQuery.png)
 
 <!-- TODO: add a link to the Learning Path for Prom+Grafana, once available -->
+
+### Running on docker for development and testing
```
Comment (cdsackes) [inline README.md]: could we add this in either a separate 'Development' section, or put it alongside the other installation instructions?

Comment (smithclay) [reply README.md]: Moved up to the installation instructions part 👍

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    image: grafana/grafana-enterprise
+    container_name: grafana
+    ports:
+      - 3000:3000
+    environment:
+      - GF_ALLOW_LOADING_UNSIGNED_PLUGINS=lightstep-metrics-datasource
+      - GF_LOG_LEVEL=debug
+      - GF_DEFAULT_APP_MODE=development
+    volumes:
+      - grafana-data-lmd:/var/lib/grafana
```
Comment (bcronin) [inline docker-compose.yml:13]: Does this need to match what's in the README? `grafana-data` versus `grafana-data-lmd`?

Comment (smithclay) [reply docker-compose.yml:13]: yes, just updated :)

**Review by bcronin — APPROVED** [review]

**Pull request merged**